modules/rest-api/src/definition-executor.ts

```diff
@@ -205,32 +215,41 @@ export class UserDefinitionExecutor implements DefinitionExecutor {
 
   async authorize(
     reqData: GeneralUserEntityRequestData,
-    session?: Nullable<Session>
+    session?: Session
   ): Promise<boolean> {
     return this.definition.authorize!(reqData, session);
   }
 
   async validate(
     reqData: GeneralUserEntityRequestData,
-    session?: Nullable<Session>
+    session?: Session
   ): Promise<void> {
     return this.definition.validate!(reqData, session);
   }
 
   async normalize(
     reqData: GeneralUserEntityRequestData,
-    session?: Nullable<Session>
+    session?: Session
   ): Promise<GeneralUserEntityRequestData> {
     return this.definition.normalize!(reqData, session);
   }
 
   async execute(
     reqData: GeneralUserEntityRequestData,
-    session?: Nullable<Session>
-  ): HandlerResult<GeneralUserEntityResponseData> {
+    session?: Session
+  ): HandlerResult<UserEntityResponseData> {
     if (reqData.method == "login") {
       return this.login(reqData.payload, session);
     }
+    if (session === undefined) {
+      const errorResult: ErrorResponseData = {
+        type: "error",
+        payload: createServerError(
+          `Execute method ${reqData.method} must needs session`
+        )
```

Comment:
createServerError should also have an error type
```suggestion
        ,'Unauthorized')
```